README_py.md

```diff
@@ -0,0 +1,26 @@
+# Welcome to a Toy PyBind11 Algorithm
+
+## Why?
+The goal of this project is to allow users to take an algorithm written in C++ and translate that into runnable Python code.
+This could allow a user to develop C++ code and run it on the EmotiBit's firmware, software, and through Python hitting three different metrics.
+
+## Installation Steps
+* Follow ['this link'](https://github.com/EmotiBit/EmotiBit_Biometric_Lib/tree/master/py) for details on how to setup a basic Anaconda environment.
+  * Everything up to downloading Spyder is a necessary step in the process. Skip downloading the Anaconda environment given, you can find the correct one in the GitHub repo below.
+* Afterwards, follow [this link](url), and import the pyExample_alg01/EmotiBit-pyenv-modern file into your Anaconda environments.
+* Install Visual Studio 2022 and update to the latest version if possible.
+
+## Steps to Run Example
+* Clone ['this'](https://github.com/EmotiBit/EmotiBit_Plugins) GitHub Repo onto your machine.
+* Open a ***Anaconda*** terminal and cd into EmotiBit_Plugins/cppExample_alg01
+* Once inside the cloned folder run the commands below:
+```bash
+mkdir build
+cd build
+cmake ..
+```
+* After this you should see a rounder.sln file inside your Visual Studio explorer. Open this file.
+  * After you open this file, navigate to the top of Visual Studio and find Build > Build Solution.
+  * This will create a rounder.pyd file in either the debug or release folder depending on your machine.
+  * Move the rounder.pyd file into the pyExample_alg01 folder.
```

Comment:
Moving files sounds tedious. Has a tendency to create multiple sources of truth.
For example, if you update the cpp algo and build, but forget to move the `pyd` file because an older `pyd` already exists in the python script folder.

Can we perhaps make the pyhon interpreter look into `Release/ ` folder for the pyd files? Can we solve this by restructuring the over folder organization? The answer maybe no, but something to think about.

Reply:
Fixing that structure now, we can keep the current structure for now. After talking we may want to change the repo structure and revisit.